ci/util/serial_interface.py

```
@@ -173,6 +200,29 @@ def _producer() -> None:
+            # Older fbuild version without reset_device — fall back to pyserial
+            try:
+                import serial as pyserial
+
+                s = pyserial.Serial(self._monitor.port, 115200, timeout=0.1)
+                s.dtr = True
+                await asyncio.sleep(0.1)
+                s.dtr = False
+                await asyncio.sleep(0.5)
+                s.close()
+                return True
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)